eng/scripts/build.ps1

```diff
@@ -1,6 +1,8 @@
 #Requires -Version 7.0
 param($filter, [switch]$clean, [switch]$vet, [switch]$generate, [switch]$skipBuild)
 
+. $PSScriptRoot\meta_generation.ps1
```

Comment:
In general I prefer forward slashes as it's more cross-platform friendly/consistent.